### DISCUSSION TITLE
Log in to Twitch | Your browser is not currently supported

### DISCUSSION THREAD

**Discussion by tankow79** [kaliiiiiiiiii/Selenium-Profiles#109] [Q&A] [answered]
Hello, is it possible to bypass the error "Your browser is not currently supported. Please use a [recommended browser](https://help.twitch.tv/s/article/supported-browsers) or learn more [here](https://help.twitch.tv/s/article/supported-browsers)." on the page (https://www.twitch.tv/login)

**Comment by ttlns** [comment]
huh no idea, I'll give it a try tho.

**Reply by tankow79** [reply]
Hello, did something happen?

**Reply by ttlns** [reply]
nope, I'll let you know when I find some time

**Reply by ttlns** [reply]
@tankow79 
We can't reproduce this.
1. Are you sure it works manually?
2. What exact OS are you on? 
3. Can you try manually on a different ip//network to ensure you're not IP-blocked?
4. Please provide a script you get blocked consistently with

**Reply by tankow79** [reply]
So it works in manual and errors "Your browser is not currently supported. Please use a [recommended browser](https://help.twitch.tv/s/article/supported-browsers) or learn more [here](https://help.twitch.tv/s/article/supported-browsers)."

no but only I will try at least driver.get('https://www.twitch.tv/login') the error "Your browser is not currently supported. Please use a [recommended browser](https://help.twitch.tv/s/article/supported-browsers) or learn more [here](https://help.twitch.tv/s/article/supported-browsers)." will appear

1 yes, because when I enter the data myself, there is no error
2 windows 10 pro 20h2
3 So I changed networks and ip it didn't help
4 
```from selenium_profiles.webdriver import Chrome
from selenium_profiles.profiles import profiles
from selenium.webdriver.common.by import By  # locate elements
from seleniumwire import webdriver

from time import sleep


username = ""
password = ""
CLIENT_ID = ""


def interceptor(request) -> str:
    if (
            request.method == 'POST'
            and request.url == 'https://passport.twitch.tv/protected_login'
    ):
        import json
        body = request.body.decode('utf-8')
        data = json.loads(body)
        data['client_id'] = CLIENT_ID
        request.body = json.dumps(data).encode('utf-8')
        del request.headers['Content-Length']
        request.headers['Content-Length'] = str(len(request.body))




profile = profiles.Android()  # or .Android
options = webdriver.ChromeOptions()
# options.add_argument('--log-level=3')
# options.add_argument('--disable-web-security')
# options.add_argument('--allow-running-insecure-content')
# options.add_argument('--lang=en')
# options.add_argument('--no-sandbox')
# options.add_argument('--disable-gpu')
# options.add_argument("--headless=new")
driver = Chrome(profile, options=options,
                uc_driver=False,
                # seleniumwire_options=True
                )

driver.request_interceptor = interceptor

driver.get('https://www.twitch.tv/login')

input("Press ENTER to exit: ")
driver.quit()  # Execute on the End!
```

**Comment by ttlns** [comment, marked as answer]
@tankow79 
Yep. now this makes more sense:
1. `--disable-web-security`, `--allow-running-insecure-content`, `--no-sandbox`, `--disable-gpu`, `--headless=new` are all flags which can make you detected
2.  using seleniumwire can be detected as well because of its not chrome-like tls fingerprint
3. Using device emulation currently is detectable, see https://github.com/kaliiiiiiiiii/Selenium-Profiles/issues/58
4. all chromedriver based libraries (like selenium & undetected-chromedriver) have leaks which can be detected.

Maybe https://github.com/kaliiiiiiiiii/Selenium-Driverless helps you.
It supports request interception without using proxy (see https://github.com/kaliiiiiiiiii/Selenium-Driverless#use-events)

**Comment by tankow79** [comment]
Thank you !
It worked for me:)
